apps/web/pages/apps/installed/[category].tsx

```diff
@@ -163,55 +215,47 @@ const IntegrationsContainer = ({
     other: FiGrid,
   };
 
+  if (!data && isLoading) {
+    return <SkeletonLoader />;
+  }
+
+  if (!data && !isLoading) {
+    return (
+      <EmptyScreen
+        Icon={emptyIcon[variant || "other"]}
+        headline={t("no_category_apps", {
+          category: (variant && t(variant).toLowerCase()) || t("other").toLowerCase(),
+        })}
+        description={t(`no_category_apps_description_${variant || "other"}`)}
+        buttonRaw={
+          <Button
+            color="secondary"
+            data-testid={`connect-${variant || "other"}-apps`}
+            href={variant ? `/apps/categories/${variant}` : "/apps/categories/other"}>
+            {t(`connect_${variant || "other"}_apps`)}
+          </Button>
+        }
+      />
+    );
+  }
+
   return (
-    <QueryCell
-      query={query}
-      customLoader={<SkeletonLoader />}
-      success={({ data }) => {
```

Comment:
Bye 👋

Reply:
Is it really better? 🤔

Reply:
Let's keep QueryCell for now. Will take all of them in another PR

Reply:
QueryCell has been un-deprecated AFAIK